### DISCUSSION TITLE
Wanting to help for the 3.0 update

### DISCUSSION THREAD

**Discussion by JanTamis** [AvaloniaCommunity/Material.Avalonia#238] [General]
Hi how can i help with the 3.0 update?

**Comment by SKProCH** [comment]
Hello, you can take any issues with `help wanted` or `up for grabs` tags. For example: 
- https://github.com/AvaloniaCommunity/Material.Avalonia/issues/233
- https://github.com/AvaloniaCommunity/Material.Avalonia/issues/234

**Reply by JanTamis** [reply]
one good to know, ill try to start with #233

**Reply by JanTamis** [reply]
please note that this is my first contribution so any tips are welcome

**Comment by JanTamis** [comment]
I get a 403 error while creating a pull request what am I doing wrong?

<img width="946" alt="image" src="https://github.com/AvaloniaCommunity/Material.Avalonia/assets/73336709/4547d574-30c1-4526-be7c-4d8cbeed18a8">

**Reply by SKProCH** [reply]
I think you should firstly create a new fork of this repo, after that push a branch to that fork and create a PR through a GitHub GUI. May be you can just make a fork and Rider do everything else for you. Basically, here should be your fork repository:   
![image](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/29896317/15c9388e-300f-477e-bf5d-bc8534d06e95)

**Reply by JanTamis** [reply]
thank you for your help :)
